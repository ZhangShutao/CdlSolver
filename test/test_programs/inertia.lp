time(0..3).
occur(on(1), 0).
occur(on(2), 2).

:- occur(X1, T), occur(X2, T), time(T), X1 != X2.
occur(X, T+1) :- time(T), time(T+1), occur(X, T), &c{occur(X, T+1)}.