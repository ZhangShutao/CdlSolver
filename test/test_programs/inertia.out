default model: 1
x:time(1),time(3),time(2),occur(on(2),2),time(0),occur(on(1),0); y:time(1),time(3),time(2),occur(on(2),2),time(0),occur(on(1),0);
Satisfiable